apps/emqx_exhook/.gitignore

```diff
@@ -25,5 +25,5 @@ data/
 *.class
 Mnesia.nonode@nohost/
 src/emqx_exhook_pb.erl
-src/emqx_exhook_v_1_hook_provider_client.erl
```

Comment:
Maybe we could move these auto-generated codes into a specific path?

Reply:
Let me try.